### ISSUE TITLE
Registration function

### ISSUE THREAD

**Issue by qy21gafy** [qy21gafy/AMOSSharedDeskFeatureBoard#8]
## User story
1. As a new user
2. I want / need to be able to register for the system with my data
3. So that I can login later and access the features of shared desk management

## Acceptance criteria
* User should have to provide certain data during registration, such as his email, name, password and company
* The provided data should be stored in the database
* After registration the user should not immediatly be able to access the system, but has to be approved first 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by qy21gafy** [comment]
Later on add a issue that prevents simple attacks such as SQL injection
